cli/sawtooth_cli/rest_client.py

```diff
@@ -82,11 +82,12 @@ def _get(self, path, **queries):
             self._base_url + path + self._format_queries(queries))
 
         # concat any additional pages of data
-        while 'next' in json_result.get('paging', {}):
-            previous_data = json_result.get('data', [])
-            code, json_result = self._submit_request(
-                json_result['paging']['next'])
-            json_result['data'] = previous_data + json_result.get('data', [])
+        if isinstance(json_result, dict):
```

Comment:
isinstance is an anti-pattern. What are the situations where a non expected type is being returned?